src/compiler/types.ts

```diff
@@ -4812,21 +4815,6 @@ export interface EmitTransformers {
     declarationTransformers: readonly TransformerFactory<SourceFile | Bundle>[];
 }
 
-export interface SourceMapSpan {

```

Comment:
This looked bad but these are all forks of ourselves: https://github.com/search?q=%2F%5Cbts%5C.SourceMapSpan%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code